### ISSUE TITLE
macOS Server start `CGO_ENABLED` compilation error

### ISSUE THREAD

**Issue by NuroDev** [porter-dev/porter#199]
Porter Version: v0.1.0-beta.3

Wanting to quickly test out using Porter but sadly when going through the guide to get started, in attempting to start the server (`porter server start`), it returns the following error on macOS:
```bash
[error] failed to initialize database, got error Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
{"level":"fatal","error":"Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub","time":"2021-01-11T09:59:37Z"}
```

**Comment by abelanger5** [comment]
I apologize, this was an oversight -- we just started using Github actions to build releases, and the `CGO_ENABLED` env flag didn't get added. 

I'll build a patch release by the end of the day. In the meantime, if you have Docker installed on your local machine, you can test it with `porter server start --driver docker`, which will run Porter in a Docker image. You can then go to `localhost:8080` to view the dashboard. 

One last note: when you run locally, the server will not be able to create resources in AWS (we're working to make this possible soon), and some other behaviors may be unpredictable.

**Comment by NuroDev** [comment]
Ah okay, no problem. I can very much sympathize. Been getting used to using GitHub Actions my self as of late and there can always  be a few things that get missed for CI/CD.

Thanks for the work around, I'll give it a try 🙂

**Comment by abelanger5** [comment]
Sounds good -- note that when running in docker your local k8s instance may or may not be reachable from the Docker container. I've been using `minikube start --driver=hyperkit`, since this assigns an address discoverable by other Docker containers. 

This is not a problem when using the "local" driver for Porter, which runs as a binary on the host machine. 

More on this here: #107

**Comment by abelanger5** [comment]
Fixed in [v0.1.0-beta.3.1](https://github.com/porter-dev/porter/releases/tag/v0.1.0-beta.3.1) via #201. Run the install scripts again to get the latest binary and run `porter version` to make sure the output is `v0.1.0-beta.3.1`.

**Comment by oskar-gmerek** [comment]
Looks like problem come again in latest version v0.13.3

```iMac-Oskar:~/Desktop/www oskar_gmerek$ porter server start                                                                                                                                       20:46:36
Set the current driver as local
Set the current host as http://localhost:8080
getting release https://github.com/porter-dev/porter/releases/download/v0.13.3/portersvr_v0.13.3_Darwin_x86_64.zip
getting release https://github.com/porter-dev/porter/releases/download/v0.13.3/portersvr_v0.13.3_Darwin_x86_64.zip
downloaded release portersvr to file /Users/x/.porter/portersvr_latest.zip
unzipping portersvr to /Users/x/.porter
getting release https://github.com/porter-dev/porter/releases/download/v0.13.3/static_v0.13.3.zip
getting release https://github.com/porter-dev/porter/releases/download/v0.13.3/static_v0.13.3.zip
downloaded release static to file /Users/x/.porter/static_latest.zip
unzipping static to /Users/x/.porter/static
panic: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub

goroutine 1 [running]:
github.com/porter-dev/porter/api/server/shared/config/loader.sharedInit()
	/go/src/github.com/docker/cli/api/server/shared/config/loader/loader.go:54 +0xf7
github.com/porter-dev/porter/api/server/shared/config/loader.init.0()
	/go/src/github.com/docker/cli/api/server/shared/config/loader/init_ee.go:14 +0x34
Failed: exit status 2```
